apps/src/code-studio/pd/application_dashboard/cohort_calculator.jsx

```diff
@@ -28,24 +28,15 @@ export default class CohortCalculator extends React.Component {
     super(props);
 
     this.state = {
-      loadingCapacity: null,
-      capacity: null,
-      loadingRegistrationCount: null,
-      registered: null
+      loadingEnrollmentCount: null,
```

Comment:
I change `loadingRegistrationCount` to `loadingEnrollmentCount` and `registered` to `enrolled` to match the function name and value returned from the server side.